.github/workflows/release.yml

```diff
@@ -33,21 +34,19 @@ jobs:
           java-version: '11'
           distribution: 'adopt'
 
-      - name: Publish release
-        uses: release-drafter/release-drafter@v5
-        with:
-          publish: true
+      - name: Create Sonatype key
+        run: base64 -d <<< $SONATYPE_SINGING_KEY_BASE64 > sonatype.gpg
         env:
-          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+          SONATYPE_SINGING_KEY_BASE64: ${{ secrets.SONATYPE_SINGING_KEY_BASE64 }}
 
-      - name: Get release tag
-        id: tag
-        uses: dawidd6/action-get-tag@v1
-#
-#      - name: Create Sonatype key
-#        run: base64 -d <<< $SONATYPE_SINGING_KEY_BASE64 > sonatype.gpg
-#        env:
-#          SONATYPE_SINGING_KEY_BASE64: ${{ secrets.SONATYPE_SINGING_KEY_BASE64 }}
+      - name: Publish artifacts
+        run: ./gradlew checkMavenCredentials publishMavenPublicationToSonatypeRepository closeAndReleaseSonatypeStagingRepository -Pversion=${{ github.event.release.tag_name }}
```

Comment:
Can we split this line in few?

Reply:
I was originally considering splitting the credential check out to a separate action, but as all the env variables need to be passed through it makes sense executing it all in one go - not sure how i can split out one execution to separate lines

Reply:
key: >
    long
    string here.